api/hiveextension/v1beta1/agentclusterinstall_types.go

```diff
@@ -283,6 +290,10 @@ type AgentClusterInstallStatus struct {
 	// ValidationsInfo is a JSON-formatted string containing the validation results for each validation id grouped by category (network, hosts-data, etc.)
 	// +optional
 	ValidationsInfo common.ValidationsStatus `json:"validationsInfo,omitempty"`
+
+	// MirrorRegistryConfigurationInfo contains the mirror registry configuration for this cluster
+	// +optional
+	MirrorRegistryConfigurationInfo *MirrorRegistryConfigurationInfo `json:"mirrorRegistryConfigurationInfo,omitempty"`
```

Comment:
Why are we setting this in status? Do you think it's very important to tell the user what they told us or are we using this in some way?

Reply:
Showing the user this information conveys two main points:

1.     Their TOML file is correctly configured and successfully fetching the passed information.
2.     When the user revisits this information, they can view the currently applied mirror registry configuration.

Reply:
>Their TOML file is correctly configured and successfully fetching the passed information

Wouldn't we fail and set `SpecSynced` condition with a helpful error of the file were malformed or we hit some other issue?

>When the user revisits this information, they can view the currently applied mirror registry configuration.

:shrug: I'm not sure it's more valuable to have multiple places where this is listed out (the original configmap and this status). If we say the spec is synced and they set the reference then that should be enough.

I would only expect something in status if it's being defined/decided by the controller rather than just repackaging information the user sent to us.